classes/Group.php

```diff
@@ -209,7 +209,7 @@ public function update($autodate = true, $null_values = false)
             Configuration::updateGlobalValue('PS_GROUP_FEATURE_ACTIVE', 1);
         }
 
-        return parent::update($autodate, $null_values);
```

Comment:
$autodate is not used anymore. Do we have a plan to remove it ?

Reply:
As it has the first place, it's hard to remove it without a big BC.